src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java

```diff
@@ -92,19 +94,32 @@ public void savesAMember() throws Exception {
     }
 
     @Test
-    public void updatesAMember() throws Exception {
-        aMember.setUsername("aNewUsername");
-        mockMvc.perform(put("/members/1")
+    public void updatesAnExistingMember() throws Exception {
+        aMember.setEmail("newExample@email.com");
+        mockMvc.perform(put(String.format("/members/%d", aMember.getId()))
             .contentType(MediaType.APPLICATION_JSON)
             .accept(MediaType.APPLICATION_JSON)
             .content(mapper.writeValueAsString(aMember)))
             .andExpect(status().isOk())
-            .andExpect(jsonPath("$.username", is(aMember.getUsername())));
+            .andExpect(jsonPath("$.email", is(aMember.getEmail())));
+    }
+
+    @Test
+    public void createsANewMemberWhenIdNotFound() throws Exception {
+        Member anotherMember = new Member();
+        anotherMember.setUsername("anotherMemberUsername");
+
+        mockMvc.perform(put(String.format("/members/%d", aMember.getId()+1))
+            .contentType(MediaType.APPLICATION_JSON)
+            .accept(MediaType.APPLICATION_JSON)
+            .content(mapper.writeValueAsString(anotherMember)))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.username", is(anotherMember.getUsername())));
     }
```

Comment:
I have no idea why I enabled this behavior. I don't see a good reason to support this. Maybe it was copy-pasta. I'm going to remove this behavior next time I work on things. Sorry for the unnecessary work